src/barril/units/_tests/test_array.py

```diff
@@ -214,12 +214,21 @@ def testDivision(unit_database_len_time):
     assert calculated1 == s1 / s2
 
 
+def testFloorDivision():
+    a = Array([3.5, 4.2], "m")
+    b = Array([100.0, 100.0], "cm")
+    assert approx((a // b).GetValues("")) == [3.0, 4.0]
```

Comment:
Why `""`? Shouldn't it be no argument at all?

Reply:
`GetValues()` works too. I used `""` because it was the explicit resulting unit (actually no unit) of the `a // b` operation.

Reply:
OK